homeassistant/components/dlna_dms/config_flow.py

```diff
@@ -183,3 +201,42 @@ async def _async_get_discoveries(self) -> list[ssdp.SsdpServiceInfo]:
             for entry in self._async_current_entries(include_ignore=False)
         }
         return [disc for disc in discoveries if disc.ssdp_udn not in current_unique_ids]
+
+
+class DlnaDmsOptionsFlowHandler(OptionsFlow):
+    """Handle a DLNA DMS options flow.
+
+    Configures the single instance and updates the existing config entry.
+    """
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize."""
+        self.config_entry = config_entry
```

Comment:
It would probably not surprise you, but oh boy we have something for this
```suggestion
class DlnaDmsOptionsFlowHandler(OptionsFlowWithConfigEntry):
    """Handle a DLNA DMS options flow.

    Configures the single instance and updates the existing config entry.
    """
```